### PR TITLE
Components part1

### PR DESCRIPTION
**Pull request by DiamondGotFenny**

### CODE REVIEW

File: components-part1/src/components/EditMenu.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+import styled from 'styled-components';
+
+const StyledMenu = styled.nav`
+  display: inline-flex;
+  flex-direction: column;
+  justify-content: center;
+  color: #fff;
+  background: #effffa;
+  transform: ${({ open }) =>
+    open ? 'translate(4rem, -13rem)' : 'translateX(9999px)'};
```
Comment (huaichaow) [inline components-part1/src/components/EditMenu.js]: show hide it on the page as well when closed

File: components-part1/src/partials/Home/Search.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+import Logo from '../../components/Logo';
+import {
+  SearchWrapper,
+  StyledHeader,
+  StyledAddMovieButton,
+  StyledSearchTitle,
+  StyledSearchContainer,
+} from '../../styledComponents/Search';
+
+const Search = (props) => {
```
Comment (huaichaow) [inline components-part1/src/partials/Home/Search.js:12]: the presence of `PropTypes` is mandatory as per the criteria.

File: components-part1/src/components/Logo.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+import styled from 'styled-components';
+
+const StyledLogo = styled.div`
+  color: #f65261;
+  height: 24px;
+  width: 151px;
+  margin-left: 20px;
+  transform: translateY(-10px);
+  & .logo_1 {
+    font-family: Montserrat;
```
Comment (huaichaow) [inline components-part1/src/components/Logo.js:12]: we can simply share some of the properties not changed in `.logo_1` and `.logo_21.

Comment (DiamondGotFenny) [reply components-part1/src/components/Logo.js:12]: fixed, have them share same properties

**Pull request merged**